lucida/calendar/Makefile

```diff
@@ -18,15 +18,15 @@ client:
 
 corenlp:
 	@if [ ! -d "lib" ]; then \
-		wget http://nlp.stanford.edu/software/stanford-corenlp-full-2015-12-09.zip; \
-		unzip stanford-corenlp-full-2015-12-09.zip; \
-		mv stanford-corenlp-full-2015-12-09 lib; \
+		wget -c http://nlp.stanford.edu/software/stanford-corenlp-full-2016-10-31.zip; \
+		unzip stanford-corenlp-full-2016-10-31.zip; \
+		mv stanford-corenlp-full-2016-10-31 lib; \
```

Comment:
@KamalGalrani, can you add `stanford-corenlp-full-2016-10-31.zip` to the lucida/calendar/.gitignore since you updated this package and we don't want to track that within github?